src/chapters/sql-intro/sql-intro.rst

```diff
@@ -1,32 +1,36 @@
 SQL Part 1 - Introduction and Azure Data Studio
 ===============================================
 
-Read the following articles, follow along where instructed:
-
-Introduction to SQL
--------------------
-
-1. `What is SQL? <https://www.youtube.com/watch?v=27axs9dO7AE>`__
-
-2. `What Is a Database? <https://www.oracle.com/database/what-is-database/>`__
+.. admonition:: Note
 
-3. `Learn to write SQL queries in 5 minutes! <https://medium.com/geekculture/writing-your-first-sql-query-610da7a5afea>`__
+   As you read through the articles in the SQL lessons, you may find that you want to try out the queries the authors describe. 
+   You have access to a database called ``AdventureWorks2019`` if you would like to try out some of these queries. 
+   Doing so is additional practice separate from the exercises.
 
-4. `Learn to write basic SQL Queries. <https://www.sqlshack.com/learn-to-write-basic-sql-queries/>`__
 
-Read the following article, skipping the installation and set up section and stopping at the Join section.  We are just looking for comprehension to help guide when completing the exercises.  As they are using SQLite, some of the terminology may be slightly different.  We are using T-SQL which uses TOP instead of LIMIT.
+Read the following articles, follow along where instructed:
```

Comment:
Question -> in other chapters we have just a period at the end.

Reply:
only in SQL part 3.  the rest seem to be colons - or they will be soon.